packages/components/ng-ovh-sso-auth/src/provider.js

```diff
@@ -398,6 +398,7 @@ export default function() {
       if (!deferredObj.logout) {
         deferredObj.logout = $q.defer();
         isLogged = false;
+        const self = this;
```

Comment:
i think you can simply use ```this```right ? :)

Reply:
haha yes. I just kept the "style" of other methods :laughing: